src/modules/dispatcher/dispatcher.c

```diff
@@ -1800,15 +1800,16 @@ static void dispatcher_rpc_add(rpc_t *rpc, void *ctx)
 {
 	int group, flags;
 	str dest;
+  str attrs;
 
 	flags = 0;
 
-	if(rpc->scan(ctx, "dS*d", &group, &dest, &flags) < 2) {
+	if(rpc->scan(ctx, "dS*d", &group, &dest, &flags, &attrs) < 3) {
```

Comment:
I think here is missing and extra `S` in scan's 2nd parameter.

Reply:
The above condition needs some update. Because attrs is optional, then it is still fine to go on with return code 2 (which represents the number of items/params scanned).

Probably return code has to be stored in a variable, e.g., then if ret<2 is error, if ret==2 then attrs.s=NULL and attrs.len=0 (or attrs=STR_NULL when declared).

Reply:
Right, I'm going to fix.